src/snes/supermetroid/addresses.js

```diff
@@ -50,4 +50,7 @@ export default {
     prRealtimeRoom: new MemState(wram.uint16Read(0xFD06), "prRealtimeRoom", '[Practice Rom] Realtime Room'),
     prLastRealtimeRoom: new MemState(wram.uint16Read(0xFD08), "prLastRealtimeRoom", '[Practice Rom] Last Realtime Room'),
     prTransitionCounter: new MemState(wram.uint16Read(0xFD10), "prTransitionCounter", '[Practice Rom] Transition Counter'),
+
+    ridleyIframeTimer: new MemState(wram.uint16Read(0x0fa0), 'ridleyIframeTimer', 'Ridley Iframe Timer'),
+    ridleyShotDamage: new MemState(wram.uint16Read(0x187A), 'ridleyShotDamage', 'Ridley Shot Damage'),
```

Comment:
These should probably be renamed to be a bit clearer.

Looking at the decomp RAM map, $0FA0 is actually the i-frame timer for any Enemy 0, and $187A lists as: "Enemy spritemap entry X position during enemy collision detection with projectiles or Samus. Also used as a mirror of projectile damage if they do collide, then later total damage."

Neither one is specific to Ridley. Coming up with better names is tough though...